config/config.yaml

```diff
@@ -0,0 +1,4 @@
+config:
+  install_tree:
+    # Completely ignoring higher-level configuration options is supported with the :: notation for keys ...
+    root:: $spack/../release
```

Comment:
This defaults to intalling into `release`.

So the intention is to use this for deployment from GitHub only? If we wanted to test things in `dev` how would we do that and make sure we don't accidentally write into `release`? We could make `release` owned by the service user and not allow group write?

Reply:
How do you want to use `dev` and `release`? This file may need to be generated depending on our intentions.

Reply:
From @aidanheerdegen : "I would say CD should install directly into pre-release and there should be a subsequent CD into release after an approval step. In some cases this would be fairly automatic/trivial, but in some cases we might need more extensive oversight, e.g. ask for some assistance from the model team. But it seems like something we should have a chat about."